app/src/main/java/com/lagradost/cloudstream3/ui/settings/SettingsFragment.kt

```diff
@@ -155,6 +156,12 @@ class SettingsFragment : Fragment() {
             return getLayoutInt() == 2
         }
 
+        // phone exclusive
+        fun isTruePhone(): Boolean {
```

Comment:
this is sooooo bad, look at Context.isTvSettings and Context.updateTv and implement this as a bool instead